PWGJE/EMCALJetTasks/AliAnalysisTaskJetChargeFlavourTemplates.cxx

```diff
@@ -63,7 +62,6 @@ AliAnalysisTaskJetChargeFlavourTemplates::AliAnalysisTaskJetChargeFlavourTemplat
   fCentMax(10),
   fJetRadius(0),
   JetChargeK(0.5),
-  MotherFraction(0.8),

```

Comment:
Same here - even if you're not going to set it via your add task, you need to initialize `ProgenetorFraction` in your constructor

Reply:
Hi Raymond, Maybe im being stupid and missing something here, But since The ProgenetorFraction is a varible for the tree output, (ie, Pt, Phi, Jet Charge) then it does not need to be initalized in the same way as MotherFraction was.
Is the confusion here becuase MotherFaction is varible set at runtime and used in the code and progenaator faction is its replacement but is instead an output which i can make a cut on after the train has run? Or am i just incorrect
Thanks for the help!

Reply:
Hi Sammy, yeah, you need to initialize it - see my new review below. I wanted to mark it at the constructor, and the `MotherFraction` line was just the easiest place